### DISCUSSION TITLE
Ventura-Monterey-OC-0.8.7-Release-Infinix-InBook-Series

### DISCUSSION THREAD

**Discussion by devboloji** [devboloji/Infinix-Hackintosh-Opencore-Guide#24] [Discuss about the release]
**This may be the final release of Infinix Hackintosh from me. Because nothing to Fix further in EFI.**
no need to update the OpenCore further.

- Fixed Fan spinning while device in sleep.
- Moved to Opencore 0.8.7 stable. 
- Changed SMBIOS from `MacbookAir9,1` to `MacbookPro16,2` for better Performance.
- Increased Vram from 3gb to 4GB.
- Moved from Metal 2 to Metal 3. Added Metal 3 support. Now the apps and games are well optimised.
- Fixed Bluetooth. From now onwards you won't see "Volume HashMismatch". Bluetooth is 100% functional for X1 i3. If you face any issue with Bluetooth or malfunction, just Open a terminal and type `sudo purge` . If it doesn't work, just reboot.
- Talking about BigSur, This EFI supports BigSur by changing the kexts in this EFI. BigSur EFI is EOL because of less users are using BigSur.
- Added PCI Stuff.
- Fully optimised whole system.
- In Ventura, There is an issue with WIFI. Just Update the Airportltlwm.kext when they release the stable. The issue is not from me.
- All Usbs are working as Windows. Now you can file transfer with Charging Type C slot as like windows.
- Fixed Type-C s. Now it's functional.
- Fixed USB2.0 & USB3.0s.
- Updated all the required kexts. 
- Fixed Camera. Now it's detected.
- Fixed Sleep. From now onwards you can put your device sleep for long hours. The sleep is working as like windows. When you put your device in sleep it consumes just 6-8 % of battery  approx. for 12 hrs.
- Now the battery backup in macOS is 6hrs. Better than windows and linux. According to my usage.
- Added FileVault Support. Now you can use our device Same as a MacBook.
- Added Boot Chime. When you start your device from shutdown, you will hear the MacBook Boot Sound. 
- Modified Boot Menu UI. The the boot menu UI is like M1.
- Increased performance of CPU.
- Optimised the power management when the device is on IDLE or running small applications i.e when you are using small applications, fan will not spin and Cpu will take low power like cpu can go to 0.3 Ghz lower when it's idle. This optimisation is better than windows power management.
- Better stop using One-key-high-dpi, it makes your device slow and lag. Just go with default 1080p resolution.
- This may be the last update.
- **If you enjoyed this project — or just feeling generous, consider buying me a coffee. Cheers! ☕ :**
# :diamond_shape_with_a_dot_inside:buy me a coffee :coffee:  Donate :  :diamond_shape_with_a_dot_inside:

<a href="https://www.paypal.me/369639/"><img src="blue.svg" height="40"></a>  
| <a><img src="darkblue.svg" height="40"></a> |
|-------------------------------------------|
|  `7095160636797@paytm` |

<hr /><em>This discussion was created from the release <a href='https://github.com/devboloji/Infinix-Hackintosh-Opencore-Guide/releases/tag/vfinal'>Ventura-Monterey-OC-0.8.7-Release-Infinix-InBook-Series</a>.</em>

**Comment by ghost** [comment]
LETS GOOO

**Reply by devboloji** [reply]
@bunnybavros how is the EFI ?😉

**Reply by miftasoft** [reply]
Works very well, thank you dev

**Reply by ghost** [reply]
They are amazing, man. This is incredibly functional and stable. Thank you!

**Comment by devboloji** [comment]
@miftasoft @bunnybavros You are welcome guys. I have been working to make this project possible since 6months. And this day it's here. If you are enjoying my project, consider donating or buy me a coffee..😉

**Reply by extasea** [reply]
Thankyou for your hardwork mate,

**Reply by devboloji** [reply]
@miftasoft @bunnybavros @extasea facing any issues with new efi?

**Reply by miftasoft** [reply]
So far I haven't found an issue

**Comment by amuejunankar** [comment]
****### Battery Backup of MacOS Monterey , OPEnCORE 8.7 EFI****

### DEVICE Name : Infinix X1 ( i5 variant )
Brightness Level -> 60% - 80%
Sound Level -> 75%

### MIX-USAGE ( 6 Hr 40 Min) 🎉

	- 2 Hrs Of Youtube ( WiFi-On ) ( Brave )
	- 1:40 Hrs Of Web Browsing ( Safari & Brave )
	- 3 Hrs Of Video Offline ( VLC )

### Which Things Consumes more battery
	
- Wi-Fi  And Bluetooth ( turn off when not needed )
- Chrome, Edge Browser ( Consumes more battery And Resources too. # USE Safari instead )
- Dynamic Wallpaper (#  Use Static Wallpaper)

### Always Wi-Fi on Mode ( 4-5 Hrs )
### Heavy Usage like Android Studio ( 2- 3 Hrs )

When the battery is at 5%, the device turns off.

# THANK YOU SO MUCH @devboloji ❤️

**Reply by ghost** [reply]
I'm confirmed that this information is true according to my real usage. @amuejunankar 🔥

**Comment by devboloji** [comment]
@amuejunankar Firstly, thank you for sharing your Feedback and Battery backup Usage. Actually you will get almost 8 hours of battery backup for Common usage. Between what is your device name? and When did your device shutdown in MIX- USAGE challenge? If you add  these details about these, These will make your review more accurate. Talking about "Which Things Consumes more battery session" you are absolutely correct in this battery consumption topic. And if you disable some settings like `Universal Control` and `always find bluetooth devices and WI-FIs` settings, it will give you more battery.
and Thank you for using this PROJECT. IF you like my project, Consider Buy me a Coffee.. 😄 🥇

**Comment by Causoares2003** [comment]
Amazing work, with minor changes I was able to enjoy this very well made EFI on my Asus laptop, worked very stable on Ventura, but I gonna use Monterey because I think its faster/better optimized on my laptop, also the new settings menu looks bad. Anyway thank you very much again.

**Comment by devboloji** [comment]
@Causoares2003 I'm happy to hear that my EFI is working for your device. Would you like to contribute anything for this project? 🥳 🎈 🎉

**Comment by extasea** [comment]
I updated the KEXT & EFI to OC 0.9.0 
The update was great, faster booting and working good except touchpad

Any advice about this to make the touchpad works?

Thanks

**Reply by devboloji** [reply]
@extasea  what about boot time? How much faster its booting?

**Reply by extasea** [reply]
I only had 8-10 seconds to boot since the power button pressed until login

**Reply by devboloji** [reply]
@extasea oww. Great. I have been working on upcoming Release of efi.  So wait for the release.

**Reply by ghost** [reply]
The booting speed was almost the same as it is on Windows!

I stopped using Hackintosh since I found out that some plugins for my music production run faster and better on Windows. But I will try to update the OpenCore and see if it improves. :)

**Reply by extasea** [reply]
Have you tried logic pro x on the hackintosh?

I loved how the hackintosh video editing using final cut pro, it way more faster than windows

**Reply by ghost** [reply]
I have never used Logic Pro before, maybe because I'm stuck with Ableton and it's hard to switch DAWs now. Haha.
*But I'll try anyway! Let's see if it better!

**Comment by amuejunankar** [comment]
So recently I found out that **HDMI** is not working. even it works great in windows.
How to solve this ?
_POV (No issue with cable and monitor)_

**Specs**:
_Infinix X1 I5 | OPENCORE 8.7 | Monterey_

**Reply by ghost** [reply]
As I have mentioned previously.
The HDMI port is not going to be fixed since Ice Lake does not support HDMI, unless you are using an adapter (USB to HDMI).

**Reply by amuejunankar** [reply]
That's sad to know.
Thank you for taking the time to reply.
